### PR TITLE
pgcenter: init at 0.5.0

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/pgcenter/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+buildGoPackage rec {
+  name = "pgcenter-${version}";
+  version = "0.5.0";
+
+  goPackagePath = "github.com/lesovsky/pgcenter";
+
+  src = fetchFromGitHub {
+    owner  = "lesovsky";
+    repo   = "pgcenter";
+    rev    = "refs/tags/v${version}";
```
Comment (worldofpeace) [inline pkgs/tools/misc/pgcenter/default.nix]:
```suggestion
    rev    = "v${version}";
```

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build pgcenter

File: pkgs/tools/misc/pgcenter/default.nix (hunk 1 above)
Comment (marsam) [reply pkgs/tools/misc/pgcenter/default.nix]: fixed, sorry for the delay

File: pkgs/tools/misc/pgcenter/deps.nix [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# file generated from Gopkg.lock using dep2nix (https://github.com/nixcloud/dep2nix)
```
Comment (worldofpeace) [inline pkgs/tools/misc/pgcenter/deps.nix]: I don't see a `Gopkg.lock` in the source repo.

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build pgcenter

**Review by worldofpeace — APPROVED** [review]
Built and executed locally without problems :+1:

**Pull request merged**